ds_judgements_public_ui/templates/includes/results_filters_inputs.html

```diff
@@ -82,6 +74,15 @@
     </fieldset>
   </div>
 </div>
+<div class="structured-search__single-field-panel">
+  <div class="structured-search__limit-to-container">
+    <fieldset>
+      <legend class="structured-search__limit-to-label">From specific courts or tribunals</legend>
+      <div class="structured-search__court-options" id="court">{% include "includes/courts_options.html" %}</div>
+    </fieldset>
+  </div>
+</div>
+{% include "includes/structured_search_additional_inputs.html" %}
```

Comment:
Just wanted to check - did this line need moving? This adds the 'neutral citation' and 'keywords' fields on the structured search page, which should show up *below* the courts there, but not show up at all in the filters on the results page.